haskell-stack/README.md

```diff
@@ -0,0 +1,28 @@
+# Haskell-stack dockerfile
+
+Creates an image with [`stack`](http://docs.haskellstack.org/en/stable/README/) installed.
+
+## Build
+
+To build, run:
+
+```
+cd haskell-stack
+docker build -t advancedtelematic/haskell-stack:1.0.2 .
+```
+
+## Usage
+
+This needs `stack setup` to be run and persisted between calls, so first up run the following:
+
+```
+docker create --name stack-cache -v /root/.stack tianon/true /bin/true 2>/dev/null || true'
+```
+
+Then use that volume for the rest of the calls like so:
+
+```
+docker run --net=host -v "$(pwd):/src" --volumes-from stack-cache advancedtelematic/haskell-stack stack
```

Comment:
Breaking the network isolation with `--net=host` should only be used in special use cases. Is there any reason for this use case?